addons/hr_holidays/tests/test_access_rights.py

```diff
@@ -782,6 +782,7 @@ def setUpClass(cls):
             'leave_validation_type': 'hr',
             'requires_allocation': 'no',
         })
+        cls.employee_emp.company_id = cls.new_company
```

Comment:
I don't get this change next to `self.employee_emp.company_id = self.user_employee.company_id` later on